test/geometry/scalingtest.cpp

```diff
@@ -12,7 +12,7 @@ TEST_CASE("Geometry: scaling a Point", "[geometry][operations]") {
     PointFixture pointFixture;
 
     Point scaledPoint;
-    scale(pointFixture.point, scalingPoint, scaledPoint);
+    scale(pointFixture.point, scalingPoint.x(), scalingPoint.y(), scaledPoint);
```

Comment:
I agree that those options are better, but I had problems when using something from type Location as scalingPoint, because the Location coordinates are micrometers, instead of doubles. It would be great if someone could fix that.